### ISSUE TITLE
Why just realms

### ISSUE THREAD

**Issue by jrcarl624** [MCBE-Utilities/JavaLink#2]
Servers have the same protocol as realms. Why not add a setting to join a server along side a realm. With this not cutting off part of the community.

**Comment by ATXLtheAxolotl** [comment]
Cuz that's a lot more work.

**Comment by jrcarl624** [comment]
> Cuz that's a lot more work.

Not really, it's actually a lot more work for realms. Because you have to do Xbox authentication to get the ip and port of the realm with the realms api. Meaning it's a easy option to add server support.

**Comment by ATXLtheAxolotl** [comment]
> > Cuz that's a lot more work.
> 
> Not really, it's actually a lot more work for realms. Because you have to do Xbox authentication to get the ip and port of the realm with the realms api. Meaning it's a easy option to add server support.

Dude do you even know how BeRP works?

**Comment by ATXLtheAxolotl** [comment]
This is just a plugin for BeRP, BeRP is directly aimed at realms, I'm saying it's a lot more work because they've already done realms. They did add an external command recently that allows external servers

**Comment by jrcarl624** [comment]
> This is just a plugin for BeRP, BeRP is directly aimed at realms, I'm saying it's a lot more work because they've already done realms. They did add an external command recently that allows external servers

Yea I took a look at it, I feel the feature should be mentioned in the description and the README
